### PR TITLE
Admin page

### LINKED ISSUE #5: Create User list component and tuit list component and other necessary buttons on the administration page to create and fetch users

### LINKED ISSUE #8: Document updated React component and API requests for admin feature

### LINKED ISSUE #15: Document the updated components and API requests

### PR DESCRIPTION
**Pull request by caijiah**
Have finished admin page and added jsdoc. Will work on tests.

### CODE REVIEW

**Comment by aliu19** [comment]
Would it be better to add some navigation so you know which part of the admin page you are in? For ex. when you are in "search users" and refresh the page, it brings you back to "users table".

File: src/components/admin/index.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+            <Tab.Container id="left-tabs-example" defaultActiveKey="users">
+                <Row>
+                    <Col sm={3}>
+                        <Nav variant="pills" className="flex-column"
+                             onSelect={k => {
+                                 setKey(k)
+                                 navigate(`/admin/${k}`)
+                             }}>
+                            <Nav.Item>
+                                <Nav.Link eventKey="users">Users Table</Nav.Link>
```
Comment (aliu19) [inline src/components/admin/index.js:60]: Would it be better to add some navigation so you know which part of the admin page you are in? For ex. when you are in "search users" and refresh the page, it brings you back to "users table".

File: src/services/tuits-service.js [hunk 2]
```diff
@@ -7,6 +7,7 @@ import axios from "axios";
 const BASE_URL = process.env.REACT_APP_BASE_URL;
```
Comment (aliu19) [inline src/services/tuits-service.js:7]: I believe admin should be able to update tuits, I don't see the PUT request here / the updated tuits are not saved when I try to?

**Review by aliu19 — APPROVED** [review]

File: src/services/tuits-service.js (hunk 2 above)
Comment (caijiah) [reply src/services/tuits-service.js:7]: This feature was at Avery's branch.

File: src/components/admin/index.js (hunk 1 above)
Comment (caijiah) [reply src/components/admin/index.js:60]: Fixed

**Review by averywpx — APPROVED** [review]

**Pull request merged**